cli/src/semgrep/commands/ci.py

```diff
@@ -602,7 +602,7 @@ def ci(
 
     # Since we keep nosemgrep disabled for the actual scan, we have to apply
     # that flag here
-    keep_ignored = not enable_nosem or output_handler.formatter.keep_ignores()
+    keep_ignored = not enable_nosem or output_handler.keep_ignores()
```

Comment:
This means that if there are multiple output formats specified, and if even one of them wants to keep ignored, then all of them need to keep ignored.

Just wanted to double-check if this is the intended behavior or not.

Reply:
This was the intended behavior but only because I couldn't see an easy way to filter things for just the output formatters. We might need to push the filtering into the output formatters later, but right now the run_scan file generates metrics off the filtered results so I didn't want to change the behavior there.

Reply:
Sounds good to keep it like this now. Let's move the discussion to linear. Can you post a comment there to remind us that we should also think about `keep_ignores` for different formats?

Reply:
Done! ([link](https://linear.app/semgrep/issue/SAF-341/multiple-output-options-simultaneously#comment-c72f69dc))